fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -241,8 +241,12 @@ def ios?
     end
 
     def supported_platforms
-      supported_platforms = build_settings(key: "SUPPORTED_PLATFORMS").split
-      supported_platforms.map do |platform|
+      supported_platforms = build_settings(key: "SUPPORTED_PLATFORMS")
+      if supported_platforms.nil?
+        UI.important("Could not read the \"SUPPORTED_PLATFORMS\" build setting, assuming that the project supports iOS only.")
+        return [:iOS]
```

Comment:
Yes, that’s correct, see https://github.com/fastlane/fastlane/blob/bebdebab4feeeec7e2799730c6f59cbd592520a6/fastlane_core/lib/fastlane_core/project.rb#L240